web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -1108,10 +1108,6 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
    * This function allows us to calculate the font size in those situations.
    */
   getKeyEmFontSize(): ParsedLengthStyle {
-    if (!this.fontSize) {
-      return new ParsedLengthStyle('0px');
-    }
-
```

Comment:
The condition is never true; the property performs a default lazy-init if necessary before returning.